src/shims/fs.rs

```diff
@@ -257,6 +264,196 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         this.try_unwrap_io_result(result)
     }
 
+    fn statx(
+        &mut self,
+        dirfd_op: OpTy<'tcx, Tag>,
+        pathname_op: OpTy<'tcx, Tag>,
+        flags_op: OpTy<'tcx, Tag>,
+        _mask_op: OpTy<'tcx, Tag>,
+        statxbuf_op: OpTy<'tcx, Tag>
```

Comment:
Might be a good idea to add comments saying what type these all have.

Reply:
Sure